src/app/archive/dune

```diff
@@ -53,6 +53,6 @@
   mina_version
   bounded_types)
  (instrumentation
-  (backend bisect_ppx))
```

Comment:
Why other two executables in the `dune` file were not affected?

Reply:
good catch! merge was incorrect.